test/support/page_objects/common.js

```diff
@@ -304,10 +304,10 @@ export default class Common {
   }
 
   findTestSubject(selector) {
-    this.debug('in findTestSubject: ' + testSubjSelector(selector));
+    this.debug(`in findTestSubject: ${selector}`);
```

Comment:
Down the road I imagine us having more compound components in the UI framework, and needing to select specific sub-components from the functional tests. I suppose there are two ways we could handle this:

 1. compound components define test-subjects for their sub-components, the implementor assigns a unique test-subject to the container, and nested selectors are used to select specific components
 1. all component components expose a `testSubjectPrefix` config that allows us to modify the selectors and select them without support for nested selectors.

I'm fine with option 2 if you think that would be easier to rationalize, especially since we could remove the black-box `testSubjSelector()` function